src/main/asciidoc/repository.adoc

```diff
@@ -142,6 +143,8 @@ A few N1QL-specific values are provided through SpEL:
  - `#n1ql.filter` in the WHERE clause adds a criteria matching the entity type with the field that Spring Data uses to store type information.
  - `#n1ql.bucket` will be replaced by the name of the bucket the entity is stored in, escaped in backticks.
  - `#n1ql.fields` will be replaced by the list of fields (eg. for a SELECT clause) necessary to reconstruct the entity.
+ - `#n1ql.delete` will be replaced by the `delete from` statement.
+ - `#n1ql.returning` will be replaced by returning clause needed for reconstructing entity.
```

Comment:
Yes if the user includes returning spel placeholder, it will return the full entity with id and cas